web-wallet/src/routes/(welcome)/setup/+page.svelte

```diff
@@ -5,6 +5,12 @@
   import Animation from "./Animation.svelte";
   import { AppAnchorButton } from "$lib/components";
   import { mdiKeyOutline, mdiRestore, mdiWalletOutline } from "@mdi/js";
+  import { onMount } from "svelte";
+  import { mnemonicPhraseResetStore } from "$lib/stores";
+
+  onMount(() => {
+    mnemonicPhraseResetStore.set([]);
```

Comment:
Is this necessary?

If yes, we need to add a test for the behaviour.

Reply:
Actually, is not! It was made redundant after moving the resetting of the store in the component's lifecycle in the Restore page.

Great catch! 🎉